Description: Bugfix for the Icinga-Web PNP Plugin
 When viewing the host performance data without opening a tab the
 URL is wrong, so no image is displayed.
 .
 This is already fixed in upstream HEAD.
Author: Markus Frosch <markus.frosch@netways.de>

Origin: other
Last-Update: 2012-05-08

--- icinga-web-1.6.2.orig/contrib/PNP_Integration/templateExtensions/pnp-host-extension.xml
+++ icinga-web-1.6.2/contrib/PNP_Integration/templateExtensions/pnp-host-extension.xml
@@ -58,7 +58,7 @@
                             <parameter name="booleanConditionField">pnp4nagios_host_image_hover</parameter>
                             <parameter name="title">Image for
                                 {host_name}</parameter>
-                            <parameter name="url"><![CDATA[/pnp4nagios/image?host=C://test/test.html&srv=_HOST_&view=0]]></parameter>
+                            <parameter name="url"><![CDATA[/pnp4nagios/image?host={host_name}&srv=_HOST_&view=0]]></parameter>
                             <parameter name="width">400</parameter>
                             <parameter name="height">400</parameter>
                         </parameter>
